### ISSUE TITLE
Images not displaying in Safari

### ISSUE THREAD

**Issue by alirussell** [KyleAMathews/react-retina-image#82]
Ive come across a weird issue in Safari (10.0.2 and possibly other versions) where the images don't display.

To reproduce it, you need to:

- navigate to http://kyleamathews.github.io/react-retina-image/ the images loads properly
- click on the `Code on Github` link
- scroll down and click on the demo link again
- The images do not display
- Refresh the page
- Images display again

demo: http://drops.tray.io/260S1G252g0y

It seems to be that the image width/height is set to 0 (if you remove it, the image is actually there), and must be something to do with loading events that safari fires when getting items from cache perhaps.

**Comment by tedlin182** [comment]
Yeah it happens in Chrome for me also, resulting in the image flashing on and off. 

Any reason why you're setting the `width` and `height` attributes directly on the `img` tag?
